framework/source/class/qx/bom/IdleCallback.js

```diff
@@ -0,0 +1,109 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2016 GONICUS GmbH, Germany, http://www.gonicus.de
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+     * Cajus Pollmeier (cajus)
+
+************************************************************************ */
+
+/**
+ * This is a cross browser wrapper for requestIdleCallback. For further
+ * information about the feature, take a look at spec:
+ * https://www.w3.org/TR/requestidlecallback/
+ *
+ * Here is a sample usage:
+ * <pre class='javascript'>var clb = function(deadline) {
+ *   while (deadline.timeRemaining() > 0) {
+ *     // ... do some last tasks
+ *   }
+ * };
+ *
+ * qx.bom.IdleCallback.request(clb, this);
+ * </pre>
+ */
+qx.Bootstrap.define("qx.bom.IdleCallback",
+{
+  extend : qx.core.Object,
+
+  statics :
+  {
+    /**
+     * The default time in ms the timeout fallback implementation uses.
+     */
+    TIMEOUT : 0,
+
+    /**
+     * The default remaining time in ms the timeout fallback implementation uses.
```

Comment:
I've read through all of the code and I'm still not sure what this is. Can the documentation provide some more detail for REMAINING and TIMEOUT here?